src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -117,4 +117,11 @@ public Conversation getConversationWithTitle(String title) {
   public void setConversations(List<Conversation> conversations) {
     this.conversations = conversations;
   }
+
+  /**
+    * Returns the number of conversations currently recorded
+    */
+  public int getConversationCount() {
```

Comment:
Is this called in the Message constructor or only when loading a User from the UserStore?

Reply:
For now, this is only called when the admin page is loaded.